### PR TITLE
Add guide to connect LORIX One gateway

### LINKED ISSUE #2614: Add LORIX One guide to gateways section
#### Summary

Add LORIX One guide to gateways section as part of #1140 

#### Why do we need this ?

Make it easier to connect popular gateways

#### What is already there? What do you see now?

https://thethingsstack.io/v3.8.1/gateways/

https://github.com/TheThingsNetwork/lorawan-stack/tree/master/doc/content/gateways

#### What is missing? What do you want to see?

...

#### How do you propose to document this?

...

#### Can you do this yourself and submit a Pull Request?

Yes

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes #2614 

#### Changes

- Add guide to connect LORIX One gateway

#### Testing

Run locally

#### Notes for Reviewers

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/lorix-one/_index.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+title: "Wifx LORIX One"
+description: ""
+---
+
+The LORIX One is a robust and professional grade outdoor LoRaWAN™ gateway in an ultra compact form factor, designed and assembled in Switzerland. The LORIX One supports ethernet, wireless, and cellular backhauls.
```
Comment (laurensslats) [inline doc/content/gateways/lorix-one/_index.md:6]:
```suggestion
The LORIX One is a robust and professional grade outdoor LoRaWAN® gateway in an ultra compact form factor, designed and assembled in Switzerland. The LORIX One supports ethernet, wireless, and cellular backhauls.
```

File: doc/content/gateways/lorix-one/_index.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+<!--more-->
+
+![LORIX One](lorix-one.png)
+
+For additional help and technical specifications, please refer to [Wifx's official documentation](https://iot.wifx.net/docs).
+
+## Requirements
+
+  1. User account on {{% tts %}} with rights to create Gateways.
```
Comment (laurensslats) [inline doc/content/gateways/lorix-one/_index.md:18]:
```suggestion
  1. User account on {{% tts %}} with rights to create Gateways
```

File: doc/content/gateways/lorix-one/_index.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+If your gateway is running legacy software, please refer to [the official documentation](https://iot.wifx.net/docs) for upgrade instructions.
+
+## Get the gateway EUI
+
+To register the gateway, you will need its Extended Unique Identifier (Gateway EUI). This can be found either on the gateway's sticker or by software in the Manager UI.
+
+### From the sticker
+
+To get the Gateway EUI from the sticker, find the MAC address printed on the sticker under the gateway. The gateway EUI corresponds to the MAC address, removing the ';' and adding FFFE in the middle.
```
Comment (laurensslats) [inline doc/content/gateways/lorix-one/_index.md:30]: ';'
```suggestion
To get the Gateway EUI from the sticker, find the MAC address printed on the sticker under the gateway. The gateway EUI corresponds to the MAC address, removing the `;` and adding `FFFE` in the middle.
```

File: doc/content/gateways/lorix-one/_index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+![LORIX One LoRa hardware page](lorix-one-lora-settings-antenna.png "LORIX One LoRa hardware page")
+
+In the **Antenna** field, select the antenna you have connected.
+
+- 2dBi is the small antenna (~20cm)
+- 4dBi is the big antenna (~40cm)
+
+> Note: if the antenna type is not configured, the packet forwarder will fail to start.
+
+### Configure the packet forwarder
```
Comment (laurensslats) [inline doc/content/gateways/lorix-one/_index.md:82]: Let's again create a sub-menu with LoRa Basics Station / UDP Packet Forwarder

Screenshot 2020-06-22 at 11 25 22

**Comment by benolayinka** [comment]
> It again shows **LoRa™ Basics Station** instead of **LoRa Basics™ Station**
> 
> Please double-check before you merge.

Yep sorry I fixed that in #2765 but I didn't pull it in here yet. It will be fixed everywhere when that PR merges.

**Comment by johanstokking** [comment]
@benolayinka please rebase and check reviewers

**Pull request merged**